coralproject/cay
# Update a search with meta fields

## What?

This should have existed a while ago probably. You can now update a search and its filters along with name, description and tag.

## How to test

- create a search
- go to list of saved searches
- click Edit
- edit the filters, name, description and/or tag
- click Save
- search should be updated now here and in the Saved Searches list
